cmake/externals/triton.cmake

```diff
@@ -102,22 +102,26 @@ else()
   set(RapidJSON_ROOT_DIR ${BINARY_DIR})
 
   set(triton_extra_cmake_args "")
-  set(triton_patch_command patch --verbose -p1 -i ${PROJECT_SOURCE_DIR}/cmake/externals/triton_cmake.patch)
+  set(triton_patch_exe patch)
   set(triton_dependencies RapidJSON)
 
-  # Patch the triton client CMakeLists.txt to fix two issues when building the python wheels with cibuildwheel, which
-  # uses CentOS 7.
-  #  1) use the full path to the version script file so 'ld' doesn't fail to find it. Looks like ld is running from the
-  #     parent directory but not sure why the behavior differs vs. other linux builds
-  #       e.g. building locally on Ubuntu is fine without the patch
-  #  2) only set the CURL lib path to 'lib64' on a 64-bit CentOS build as 'lib64' is invalid on a 32-bit OS. without
-  #     this patch the build of the third-party libraries in the triton client fail as the CURL build is not found.
-
-  endif() #if (WIN32)
+endif() #if (WIN32)
 
 # Add the triton build. We just need the library so we don't install it.
-#
 set(triton_VERSION_TAG r23.05)
+
+# Patch the triton client CMakeLists.txt to fix issues when building the linux python wheels with cibuildwheel, which
+# uses CentOS 7.
+#  1) use the full path to the version script file so 'ld' doesn't fail to find it. Looks like ld is running from the
+#     parent directory but not sure why the behavior differs vs. other linux builds
+#       e.g. building locally on Ubuntu is fine without the patch
+#  2) only set the CURL lib path to 'lib64' on a 64-bit CentOS build as 'lib64' is invalid on a 32-bit OS. without
+#     this patch the build of the third-party libraries in the triton client fail as the CURL build is not found.
+#
+# Also fix a Windows issue where the security tools complain about warnings being disabled by increasing the
+# warning level for triton/src/c++/library. Was /W0. Patched to /W3.
+set(triton_patch_command ${triton_patch_exe} --verbose -p1 -i ${PROJECT_SOURCE_DIR}/cmake/externals/triton_cmake.patch)
```

Comment:
why does this patch command line not work for this case?
https://github.com/microsoft/onnxruntime-extensions/blob/4842e9d6ae97e907830b2b32502ae275f1b62e68/cmake/externals/sentencepieceproject.cmake#L13